Javatrix.java

```diff
@@ -481,5 +481,27 @@ public Javatrix times(double scalar)
         }
         return actual;
     }
+
+    /**
+     * Matrix transpose.
+     *
+     * @return A'
+     */
+    public Javatrix transpose()
+    {
+        int rows = this.getColumnDimension();
+        int cols = this.getRowDimension();
+        Javatrix actual = new Javatrix(rows, cols);
+        double a;
+        for (int i = 0; i < rows; i++)
+        {
+            for (int j = 0; j < cols; j++)
+            {
+                a = this.get(j, i);
```

Comment:
If you look at the code, you'll see I'm not actually overriding elements. I'm getting from one matrix object and putting into a different one.